### ISSUE TITLE
create standard python distribution

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#6]
```
http://docs.python.org/dist/dist.html

After that is done we can add our code to the python index, cheeseshop.
```

Original issue reported on code.google.com by `tom.brow...@gmail.com` on 22 Apr 2007 at 4:05

**Comment by GoogleCodeExporter** [comment]
```
Here are some URLs I looked at while making our source distribution and making 
it
easier to run our tests:

http://somethingaboutorange.com/mrl/projects/nose/
http://pycheesecake.org/wiki
http://pycheesecake.org/wiki/PythonTestingToolsTaxonomy
```

Original comment by `tom.brow...@gmail.com` on 29 Apr 2007 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
rev 229 added better support for nose and cheesecake (where do they get these 
names?)

I'll cut the b7 release after I get tom.brown.code-stoptime_object
tom.brown.code-utf8_weaknesses reviewed and submitted.
```

Original comment by `tom.brow...@gmail.com` on 14 May 2007 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
I created 1.0.7 source dist. I needed to rename transitfeed-1.0.7-windows to
transitfeed-windows-1.0.7 so it would be alphabetically after the source dist 
file
name. easy_install picks the first archive file listed on the page and
code.google.com sorts by file name. Ugly hack but `easy_install transitfeed` now
installs something. Unfortunately the install is broken. See issue 24.
```

Original comment by `tom.brow...@gmail.com` on 23 May 2007 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed by tom.brown.code-setup.py-fixes in transitfeed 1.0.8. Installs 
with
easy_install should now work and not litter the disk with extra files.
```

Original comment by `tom.brow...@gmail.com` on 30 May 2007 at 7:43
- Changed state: **Fixed**
